src/EVault/modules/Liquidation.sol

```diff
@@ -125,9 +122,13 @@ abstract contract LiquidationModule is ILiquidation, Base, BalanceUtils, Liquidi
         // Compute discount
 
         uint256 discountFactor = liquidityCollateralValue * 1e18 / liquidityLiabilityValue; // discountFactor = health score = 1 - discount
-
-        if (discountFactor < 1e18 - MAXIMUM_LIQUIDATION_DISCOUNT) {
-            discountFactor = 1e18 - MAXIMUM_LIQUIDATION_DISCOUNT;
+        {
+            uint256 minDiscountFactor;
+            unchecked {
+                // discount <= config scale, so discount factor >= 0
+                minDiscountFactor = 1e18 - uint256(1e18) * vaultStorage.maxLiquidationDiscount.toUint16() / CONFIG_SCALE;
```

Comment:
nit: this casting of 1e18 looks weird. maybe better to cast the uint16 value?
`uint256(vaultStorage.maxLiquidationDiscount.toUint16())`

Reply:
I created a constant to get rid of the weirdness, but it was too long or too ugly. putting it on the config is a long bracket which I find less readable